awx/settings/defaults.py

```diff
@@ -1140,12 +1140,12 @@ def IS_TESTING(argv=None):
         'awx.main': {
             'handlers': ['null']
         },
-        'awx.main.commands.run_callback_receiver': {
-            'handlers': ['callback_receiver'],
-        },
         'awx.main.dispatch': {
             'handlers': ['dispatcher'],
         },
+        'awx.main.dispatch.worker.callback': {
```

Comment:
`./installer` and `./tools/clusterdevel` may need a bit of extra work since this is renamed:

![image](https://user-images.githubusercontent.com/214912/47100981-d6316c00-d206-11e8-826c-8fcee2d60224.png)

Reply:
We went back to the old run_callback_receiver logger, so I think this actually winds up being more simple. This PR will be going back to its form before the last update. Unfortunately, I force pushed changes, so that state got lost in the actual git history.

![screen shot 2018-10-17 at 12 31 08 pm](https://user-images.githubusercontent.com/1385596/47101656-aa16ea80-d208-11e8-8da2-235708e3da0a.png)